tests/integration/cass_integration_test.go

```diff
@@ -0,0 +1,168 @@
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	chartName  = "cass-operator"
+	chartPath  = "../../charts/k8ssandra/charts/cass-operator"
+	configPath = "C:\\Users\\Jeff Banks\\.kube\\config"
+
+	crdClusterYaml = "../../charts/k8ssandra-cluster/Chart.yaml"
+
+	defaultNamespace        = "default"
+	deleteCRDTimeoutSeconds = 15
+
+	k8ssandraNamespace  = "k8ssandra-ns"
+	operatorReleaseName = "k8ssandra-1"
+	clusterReleaseName  = "cluster-1"
+	clusterName         = "dc1"
+)
+
+// TestCassOperator performs basic installation of k8ssandra operator and c* cluster using setup & teardown.
+func TestCassOperatorAndClusterInstall(t *testing.T) {
+
+	kubeOptions := k8s.NewKubectlOptions("", "", k8ssandraNamespace)
+	helmOptions := &helm.Options{KubectlOptions: kubeOptions}
+
+	fmt.Println("helmOptions namespace is", helmOptions.KubectlOptions.Namespace)
+	require.NotNil(t, helmOptions)
+
+	// operator installation
+	installOperator(t, helmOptions, operatorReleaseName)
+	lookupResult := util.LookupCRDByName(t, helmOptions, "cassandradatacenters.cassandra.datastax.com")
+	require.Equal(t, "customresourcedefinition.apiextensions.k8s.io/cassandradatacenters.cassandra.datastax.com",
+		lookupResult)
+	// todo: adding verifications of operator installation
+
+	// cluster installation
+	installCluster(t, helmOptions, clusterReleaseName)
+	// todo: adding verifications of cluster installation
+
+}
+
+// installCluster function for installation of the cluster with basic assertions
+func installCluster(t *testing.T, helmOptions *helm.Options, releaseName string) {
+	clusterInstallPreconditions(t, helmOptions, clusterReleaseName)
+	util.Annotate(t, helmOptions, clusterReleaseName)
+	isClusterInstalled := install(t, helmOptions, clusterReleaseName, "k8ssandra/k8ssandra-cluster")
+	require.True(t, isClusterInstalled)
+	assert.True(t, repoUpdate(t, helmOptions))
+}
+
+// installOperator function for installation of the operator with basic assertions
+func installOperator(t *testing.T, helmOptions *helm.Options, releaseName string) {
+
+	operatorInstallPreconditions(t, helmOptions, operatorReleaseName)
+	util.Annotate(t, helmOptions, operatorReleaseName)
+	isOperatorInstalled := install(t, helmOptions, operatorReleaseName, "k8ssandra/k8ssandra")
+	require.True(t, isOperatorInstalled)
+	assert.True(t, repoUpdate(t, helmOptions)) // cass-operator install
+}
+
+// clusterInstallPreconditions provides test cleanup and preconditions prior to test function execution
+func clusterInstallPreconditions(t *testing.T, helmOptions *helm.Options, releaseName string) {
+
+	assert.NotNil(t, helmOptions.KubectlOptions)
+	assert.NotNil(t, helmOptions)
+
+	util.CreateNamespace(t, helmOptions)
+	clusterIdentity := util.CreateClusterIdentity(t, helmOptions.KubectlOptions, clusterName, releaseName)
+
+	cleanupCluster(t, helmOptions, releaseName, clusterIdentity)
+
+	repoAdd(t, helmOptions, "k8ssandra", "https://helm.k8ssandra.io/")
```

Comment:
(comment not specific to this line only) This forces the installation to target the repository version? Including the repoUpdates and such. I would assume the integration / e2e-test is something we would want to test with the local versions - without any remote component, so as to verify the changes we've made in a PR for example.

Reply:
@burmanm this is a good question.
@jsanda this may be better answered by you.  I'm thinking this is a test run by our CI/CD pipeline with access to the repository version.  How do we want to align the versions with the local test executions?

Reply:
I think that the tests should be running against the local chart versions.

Suppose I submit a PR with some chart changes. We want tests running against my changes to verify that there are no regressions.